### PR TITLE
Harmonize the help output

### PR DESCRIPTION
**Pull request by Hoverbear**
## What have you changed? (mandatory)

Tried to harmonize the `--help` output of all tools and use a common style/tense

## What are the type of the changes? (mandatory)

- Improvement (non-breaking change which is an improvement to an existing feature)

## How has this PR been tested? (mandatory)

CI

## Does this PR affect documentation (docs) update? (mandatory)

Nope

## Does this PR affect tidb-ansible update? (mandatory)

Nope

### CODE REVIEW

File: src/bin/tikv-ctl.rs [hunk 1]
```diff
@@ -950,114 +950,115 @@ impl DebugExecutor for Debugger {
 fn main() {
     vlog::set_verbosity_level(1);
 
-    let raw_key_hint: &'static str = "raw key (generally starts with \"z\") in escaped form";
+    let raw_key_hint: &'static str = "Raw key (generally starts with \"z\") in escaped form";
     let version_info = util::tikv_version_info();
 
     let mut app = App::new("TiKV Ctl")
```
Comment (BusyJay) [inline src/bin/tikv-ctl.rs]: Control seems better.

File: src/bin/tikv-ctl.rs [hunk 2]
```diff
@@ -950,114 +950,115 @@ impl DebugExecutor for Debugger {
...
-        .setting(AppSettings::AllowExternalSubcommands)
+        .about("A distributed transactional key value database powered by Rust and Raft")
```
Comment (BusyJay) [inline src/bin/tikv-ctl.rs]: `tikv-ctl` may be just a tool.

File: src/bin/tikv-ctl.rs [hunk 3]
```diff
@@ -1075,13 +1076,13 @@ fn main() {
...
                                 .takes_value(true)
-                                .help("set the region id, if not specified, print all regions."),
+                                .help("Set the region id, if not specified, print all regions"),
                         )
                         .arg(
                             Arg::with_name("skip-tombstone")
                                 .long("skip-tombstone")
                                 .takes_value(false)
-                                .help("skip tombstone region."),
+                                .help("Skip tombstone region"),
```
Comment (BusyJay) [inline src/bin/tikv-ctl.rs]:
```suggestion
                                .help("Skip tombstone regions"),
```

File: Cargo.toml [hunk 4]
```diff
@@ -1,8 +1,11 @@
 [package]
 name = "tikv"
 version = "3.0.0-beta"
-edition = "2018"
+authors = ["The TiKV Authors"]
+description = "A distributed transactional key-value database powered by Rust and Raft"
+license = "Apache-2.0"
 keywords = ["KV", "distributed-systems", "raft"]
+edition = "2018"
```
Comment (zhangjinpeng1987) [inline Cargo.toml]: Should we update it to 2019?

Comment (Hoverbear) [reply Cargo.toml]: 2018 is the latest available. :)

**Comment by breeswish** [comment]
@hicqu PTAL

**Pull request merged**